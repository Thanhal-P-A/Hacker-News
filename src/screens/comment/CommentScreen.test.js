import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import {Provider} from 'react-redux';
import {store} from '../../../App';

import Comment from '.';

const route = {
  params: {
    idArray: [123, 456],
    storyItem: {id: 789},
  },
};

it('Comment screen renders correctly', () => {
  const HomeScreen = renderer
    .create(
      <Provider store={store}>
        <Comment route={route} />
      </Provider>,
    )
    .toJSON();
  expect(HomeScreen).toMatchSnapshot();
});
