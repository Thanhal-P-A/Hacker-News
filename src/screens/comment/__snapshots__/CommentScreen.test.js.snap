// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment screen renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "#ED642E",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "flexDirection": "row",
        "padding": 5,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#707070",
        }
      }
    >
       â–² 
    </Text>
    <View
      style={
        Object {
          "width": "90%",
        }
      }
    >
      <View
        accessibilityState={
          Object {
            "disabled": true,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#000000",
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#707070",
                "fontSize": 12,
              }
            }
          >
             (undefined) 
          </Text>
        </Text>
      </View>
      <View
        style={
          Object {
            "flexDirection": "row",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#707070",
              "fontSize": 12,
            }
          }
        >
          undefined points by undefined Invalid date
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "flex": 1,
        "justifyContent": "center",
        "padding": 10,
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#000000",
          }
        }
      >
        Please check your internet connection and retry
      </Text>
    </View>
  </View>
</RCTSafeAreaView>
`;
